poolapi/apijson.go

```diff
@@ -0,0 +1,41 @@
+package poolapi
+
+type Response struct {
+	Status  string      `json:"status"`
+	Message string      `json:"message"`
+	Data    interface{} `json:"data,omitempty"`
```

Comment:
Sure. interface{} would be the best option here on the server's type.  I still think it should be unexported as we don't expect clients to use it.